core/fx-orm.js

```
@@ -0,0 +1,939 @@
+        this.data = null;
+        this.loaded = false;
+        this.loading = false;
+        this.dirty = false;
+
+        // Start lazy loading immediately but don't block
+        this.startLazyLoad();
+
+        return new Proxy(this, {
+            get: (target, key) => this.handleGet(target, key),
+            set: (target, key, value) => this.handleSet(target, key, value)
+        });
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)